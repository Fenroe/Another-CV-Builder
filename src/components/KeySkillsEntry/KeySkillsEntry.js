function KeySkillsEntry() {
    return (
        <div>
            <fieldset>
                <legend>Key Skills</legend>
                <button>Add Skill</button>
            </fieldset>
        </div>
    )
}

export default KeySkillsEntry;
