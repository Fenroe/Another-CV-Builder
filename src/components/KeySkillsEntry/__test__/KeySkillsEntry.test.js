import { render, cleanup } from "@testing-library/react";
import React from "react";
import "@testing-library/jest-dom";
import KeySkillsEntry from "../KeySkillsEntry";

afterEach(cleanup);

it('renders', () => {
    const {asFragment} = render(<KeySkillsEntry />);
    expect(asFragment()).toMatchSnapshot();
});
