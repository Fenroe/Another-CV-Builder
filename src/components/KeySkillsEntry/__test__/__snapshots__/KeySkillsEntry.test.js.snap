// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <fieldset>
      <legend>
        Key Skills
      </legend>
      <button>
        Add Skill
      </button>
    </fieldset>
  </div>
</DocumentFragment>
`;
